client/client.go

```diff
@@ -118,8 +119,13 @@ func (c *Client) SendRequest(method string, rawURL string, data url.Values,
 
 	req.SetBasicAuth(c.basicAuth())
 
-	// E.g. "User-Agent": "twilio-go/1.0.0 (go1.16)"
-	userAgent := fmt.Sprint("twilio-go/", LibraryVersion, " (", goVersion, ")")
+	// E.g. "User-Agent": "twilio-go/1.0.0 (darwin amd64) go/go1.17.8"
+	userAgent := fmt.Sprintf("twilio-go/%s (%s %s) go/%s", LibraryVersion, runtime.GOOS, runtime.GOARCH, goVersion)
```

Comment:
Think we should try to stick to the `...[lang]/[version_number]` format within the user agent